client/src/screens/group.screen.js

```diff
@@ -368,6 +429,8 @@ const joinGroupMutation = graphql(JOIN_GROUP_MUTATION, {
           });
           // add new data to the cache
           data.user.groups.push(addUserToGroup);
+          data.user.schedules = _.merge(data.user.schedules, addUserToGroup.schedules);
```

Comment:
yeah this is good - don't modify the list in place cause weird shit might happen